src/demo-app/a11y/a11y.html

```diff
@@ -1,9 +1,12 @@
 <h1>Accessibility Demo</h1>
 
 <nav>
+  <a (click)="skipNavigation()" href="javascript:void(0)">Skip navigation</a>
```

Comment:
Changed to use `routerLink` with current path and it works. Removed `javascript:void(0)`